### PR TITLE
Fix #320 by adding a parameter to set the percentile to plot

### LINKED ISSUE #320: TelemetryWrapper should accept a param to set the percentile to plot
I'm imagining "param.percentile = 95" would let me plot the 95th percentile of a particular telemetry measurement instead of the default value of 50.

### PR DESCRIPTION
**Pull request by past**
This was pretty straightforward.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-dashboard/321)

### CODE REVIEW

File: wrapper/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Where `params` is:
 * `sensibleCompare` - A boolean dictating whether or not you want to reduce the compared values to just the ones that you are likely to care about (default: `true`)
 * `keyLimit` - a positive integer limiting the number of a keyed measure's measures to be plotted, ordered by number of submissions (default: `4`)
 * `evoVersions` - A number telling us how many versions back to look. If > 0, we will ignore trim, compare, and sensibleCompare and show an evolution instead of a histogram (default: `0`)
+* `percentile` - A number telling us the value below which a given percentage of measurements may be found (default: `50`)
```
Comment (chutten) [inline wrapper/README.md]: Should make some mention of how this is only valid when evoVersions is > 0

File: wrapper/telemetry-wrapper.js [hunk 2]
```diff
@@ -363,8 +364,11 @@ window.TelemetryWrapper.go = function (params, element) {
       ));
       valuesArePercent = true;
     } else {
-      yLabel = evolutions[0].description + ' - medians'; // i18n?
-      valueses = evolutions.map(evo => evo.percentiles(50));
+      if (params.percentile) {
+        percentileLabel = ' - ' + params.percentile + 'th percentile'; // i18n?
+      }
+      yLabel = evolutions[0].description + percentileLabel;
+      valueses = evolutions.map(evo => evo.percentiles(params.percentile || 50));
```
Comment (chutten) [inline wrapper/telemetry-wrapper.js]: Default parameters are set in setDefaultParams towards the bottom of the file. You can map 50 to "medians" to have a friendly view.

**Comment by past** [comment]
I was already mapping 50 to 'medians' in evoTime(), I hope that is what you  meant.

**Review by chutten — APPROVED** [review]
So sorry for the delay! This dropped off my radar completely

**Pull request merged**